src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml.cs

```diff
@@ -549,5 +550,43 @@ internal void Dispose()
         {
             viewModel.RequestShowFileDialog -= OnRequestShowFileDialog;
         }
+
+        int getExtraLeftSpace(int confidenceLevel)
+        {
+            int value = 16;
```

Comment:
Is this function for the slider to move the pointer? Doesn't WPF have a slider component we can use?